scripts/plots/global_ens/exevs_global_ens_headline_gefs_grid2grid_plots.sh

```diff
@@ -3,24 +3,25 @@
 # Purpose:  1. Setup the environment parameter for running GEFS headline plotting script
 #           2. Run plotting script $USHevs/global_ens/evs_global_ens_headline_plot.sh
 #  
-#  Note: 1. The headline score needs the stat files over 0ne year + 17 days 
-#           (this year + next 17 days of Jan) 
-#        2. This job is run on Jan 17 of each year                 
+#  Note: 1. The headline score needs the stat files over 0ne year + 16 days 
```

Comment:
@BinbinZhou-NOAA , can you correct that "0" (zero) to "O" (capital O) instead? Nitpicky, but would be a good correction.